pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/notification/PillarboxMediaDescriptionAdapter.kt

```diff
@@ -86,8 +86,10 @@ class PillarboxMediaDescriptionAdapter(
             outHeight = imageMaxHeight
         }
         runBlocking(Dispatchers.IO) {
-            val bitmap = BitmapFactory.decodeStream(imageUrl.openStream(), null, opts)
-            bitmap?.let {
+            val result = runCatching {
+                BitmapFactory.decodeStream(imageUrl.openStream(), null, opts)
+            }
```

Comment:
Not linked directly to this PR, but shouldn't we close the stream?
```suggestion
            val result = runCatching {
                imageUrl.openStream().use {
                    BitmapFactory.decodeStream(it, null, opts)
                }
            }
```